json-body-transformer/src/test/java/com/ninecookies/wiremock/extensions/JsonBodyTransformerTest.java

```diff
@@ -309,11 +333,21 @@ public void doesNotApplyGlobally() throws Exception {
 		wireMockServer.verify(postRequestedFor(urlEqualTo(REQUEST_URL)));
 	}
 
-	@Test
-	public void injectRandom() {
+	@DataProvider
+	private Object[][] randomFormats() {
+		return new Object[][] {
+				{ "$(!Random)" },
+				{ "$(!Random.1)" },
+				{ "$(!Random.abc)" },
+				{ "$(!Random.ABC)" },
+		};
+	}
+
+	@Test(dataProvider = "randomFormats")
+	public void injectRandom(String format) {
 		wireMockServer.stubFor(post(urlEqualTo(REQUEST_URL))
 				.willReturn(aResponse().withStatus(200).withHeader("content-type", CONTENT_TYPE)
-						.withBody("{\"randomNumber\":\"$(!Random)\", \"got\":\"it\"}")
+						.withBody("{\"randomNumber\":\"" + format + "\", \"got\":\"it\"}")
```

Comment:
In fact different. UUID is no longer a UUID if something is appended. Random (which is a number) is no longer a number. The suffix is not supported for the `$(!Instant)` and `$(!Timestamp)` keywords (as they refer to the current time, which will be the same regardless of how many properties are using a current time stamp... ;)) and for the computed times the same rules apply thus also no suffix is supported / required.